### PR TITLE
Adds required label to each field that fails validation instead of on…

### PR DESCRIPTION
**Pull request by benRmc**
…ly showing it on the first one.

**Description**

**Issues that should be CLOSED by merge of this PR:**
* Fixes #

**Related issues that should be LINKED to this PR:**
* Connects #

### CODE REVIEW

File: traveler-ios-ui/TravelerKitUI/ViewControllers/BookingQuestionsViewController.swift [hunk 1]
```diff
@@ -21,7 +21,7 @@ class BookingQuestionsViewController: UIViewController {
     var bookingForm: BookingForm?
     weak var delegate: BookingQuestionsViewControllerDelegate?
 
-    private var error: Error?
+    private var errorsDic: [IndexPath: Error?] = [:]
```
Comment (OmarPadierna) [inline traveler-ios-ui/TravelerKitUI/ViewControllers/BookingQuestionsViewController.swift]: Swift - No abbreviations.

Comment (OmarPadierna) [inline traveler-ios-ui/TravelerKitUI/ViewControllers/BookingQuestionsViewController.swift]: Error shouldn't be optional.

File: traveler-ios-ui/TravelerKitUI/ViewControllers/BookingQuestionsViewController.swift [hunk 2]
```diff
@@ -154,20 +154,30 @@ extension BookingQuestionsViewController: FormViewDelegate {
...
         case .none:
             delegate?.bookingQuestionsViewControllerDidCheckout(self)
-        case .some(.invalidAnswer(let groupIndex, let questionIndex, _)):
-            self.error = errors?.first
-
-            let indexPath = IndexPath(item: questionIndex, section: groupIndex)
-            formView.reloadFields(at: [indexPath])
-            formView.scrollToField(at: indexPath, animated: true)
+        case .some(.invalidAnswer):
+            if let errors = errors {
```
Comment (ata-n) [inline traveler-ios-ui/TravelerKitUI/ViewControllers/BookingQuestionsViewController.swift]: This is an incorrect pattern. Youre switching from `errors.first` and then catching ALL errors

**Comment by OmarPadierna** [comment]
Closing PR because this issue is no longer being considered.

**Pull request closed without merging**